CHANGELOG.md

```diff
@@ -24,9 +26,11 @@ This beta release includes a feature with a known issue: the `within` expression
  - Fixed an issue where a `LineDasharray` value of `[1, 0]` resulted in hairline gaps. ([#16202](https://github.com/mapbox/mapbox-gl-native/pull/16202))
  - Fixed Proguard configuration to prevent obfuscation of telemetry event classes. ([#195](https://github.com/mapbox/mapbox-gl-native-android/pull/195))
  - Fixed a crash that might occur during EGL window surface creation. ([#278](https://github.com/mapbox/mapbox-gl-native-android/pull/278))
+ - Fixed a bug that wrongly generated glyphs when high contrast accessibility mode was active by using RGB values instead of alpha channel. ([#289](https://github.com/mapbox/mapbox-gl-native-android/pull/289))
 
 ### Other changes
  - Updated mapbox-events-android to v4.7.3. ([#201](https://github.com/mapbox/mapbox-gl-native-android/pull/201))
+ - Revert "Transition JobIntentService to WorkManager in telemetry(#453).  ([#467](https://github.com/mapbox/mapbox-events-android/pull/467))
```

Comment:
Since this change (this telemetry upgrade) was never released in a stable release, we do not need to mention the revert in changelog.md. In other words, changes between in pre-releases don't need to be in the changelog.md file.